docs/sources/administration/plugin-management/index.md

```diff
@@ -33,19 +37,19 @@ Use panel plugins when you want to:
 
 ## Data source plugins
 
-Data source plugins add support for new databases, such as [Google BigQuery](/grafana/plugins/grafana-bigquery-datasource).
+Data source plugins add support for new databases, such as [Google BigQuery](/grafana/plugins/grafana-bigquery-datasource), [Zendesk](https://grafana.com/grafana/plugins/gabrielthomasjacobs-zendesk-datasource/) and [NetData](https://grafana.com/grafana/plugins/netdatacloud-netdata-datasource/) Data source plugins.
 
 Data source plugins communicate with external sources of data and return the data in a format that Grafana understands. By adding a data source plugin, you can immediately use the data in any of your existing dashboards.
 
-Use data source plugins when you want to import data from external systems.
+Use data source plugins when you want to import data from external or 3rd party systems.
 
 ## App plugins
 
-Applications, or _app plugins_, bundle data sources and panels to provide a cohesive experience, such as the [Zabbix](/grafana/plugins/alexanderzobnin-zabbix-app) app.
+Applications, or _app plugins_, bundle data sources and panels to provide a cohesive experience, such as the [Zabbix](/grafana/plugins/alexanderzobnin-zabbix-app), [Performance Co-Pilot](https://grafana.com/grafana/plugins/performancecopilot-pcp-app/) and [cPacket Utility Bundle](https://grafana.com/grafana/plugins/cpacket-cclearutilitybundle-app/) apps.
```

Comment:
This is generally a bad description of apps given theres no requirement to bundle anything... which I know is outside the scope of these changes, BUT technically, the cPacket one doesn't include a data source (the others do) so unless we change the description, we should change the example.

Reply:
Same intention as mentioned above in Data Sources. 

Since I am not too expert with Apps so I can simply undo the changes here.